src/olympia/editors/utils.py

```diff
@@ -207,43 +207,69 @@ def render_flags(self, record):
         return super(AutoApprovedTable, self).render_flags(
             record.current_version)
 
-    def render_addon_name(self, record):
-        url = reverse('editors.review', args=[record.slug])
+    def render_addon_name(self, record, url=None):
```

Comment:
I don't suppose it matters either way, but nit suggestion: allow `args` to be defined (i.e. `def render_addon_name(self, record, reverse_args=None): \n if not reverse_args: \n reverse_args = [record.slug] ... `) instead of overriding the whole url?

Reply:
IMHO that's not better, I find it a little more cryptic

Reply:
I think I agree with @diox here. I think it'd be more reasonable to just use a completely different URL than just different arguments for the add-on link.